tests/foreman/cli/test_computeresource_vmware.py

```diff
@@ -11,158 +11,181 @@
 
 :CaseImportance: High
 
+:CaseAutomation: Automated
+
 :Upstream: No
 """
 from fauxfactory import gen_string
 import pytest
+from wait_for import wait_for
+from wrapanapi import VMWareSystem
 
-from robottelo.cli.factory import make_compute_resource
-from robottelo.cli.org import Org
 from robottelo.config import settings
 from robottelo.constants import FOREMAN_PROVIDERS
 
 
 @pytest.fixture(scope='module')
-def vmware(module_org, module_location):
-    vmware = type('vmware', (object,), {})()
-    vmware.org = module_org
-    vmware.loc = module_location
-    Org.add_location({'id': vmware.org.id, 'location-id': vmware.loc.id})
-    vmware.vmware_server = settings.vmware.vcenter
-    vmware.vmware_password = settings.vmware.password
-    vmware.vmware_username = settings.vmware.username
-    vmware.vmware_datacenter = settings.vmware.datacenter
-    vmware.vmware_img_name = settings.vmware.image_name
-    vmware.vmware_img_arch = settings.vmware.image_arch
-    vmware.vmware_img_os = settings.vmware.image_os
-    vmware.vmware_img_user = settings.vmware.image_username
-    vmware.vmware_img_pass = settings.vmware.image_password
-    vmware.vmware_vm_name = settings.vmware.vm_name
-    return vmware
-
-
-@pytest.mark.tier1
-def test_positive_create_with_server(vmware):
-    """Create VMware compute resource with server field
-
-    :id: a06b02c4-fe6a-44ef-bf61-5a28c3905527
-
-    :customerscenario: true
-
-    :expectedresults: Compute resource is created, server field saved
-        correctly
-
-    :BZ: 1387917
-
-    :CaseAutomation: Automated
-
-    :CaseImportance: Critical
-    """
-    cr_name = gen_string('alpha')
-    vmware_cr = make_compute_resource(
+def module_vmware_cr(module_provisioning_sat, module_sca_manifest_org, module_location):
+    vmware_cr = module_provisioning_sat.sat.cli.ComputeResource.create(
```

Comment:
Can we move this fixtures to global pytest_fixtures?

Reply:
Updated, created `pytest_fixtures/component/provision_vmware.py`